ERCS/erc-9999.md

```diff
@@ -0,0 +1,397 @@
+---
+eip: 9999
+title: Minimal Upgradeable Proxies
+description: Minimal upgradeable proxies with immutable arguments and support for onchain implementation queries
+author: Atarpara (@Atarpara), JT Riley (@jtriley-eth), xiaobaiskill (@xiaobaiskill), Vectorized (@Vectorized)
+discussions-to: https://ethereum-magicians.org/t/minimal-upgradeable-proxies/20868
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7760-minimal-upgradeable-proxies/20868
```
Updated topic title.

The discussions topic can just contain a link to the PR.  There isn't a need to replicate the ERC text (as this may differ).